packages/docs/components/tag.md

```diff
@@ -76,35 +76,31 @@ Use Tags as descriptors paired with a particular object. These descriptors may b
           <td>
             Soyuz
             <ul class="ods-tag--list">
-                <li class="ods-tag">Roscosmos</li>
```

Comment:
Swap to acronyms to account for padding space?

Reply:
Yeah, we only have so much space in that second column so I made some minor tweaks based on direction from @joejezowski-okta. Starting next week we should resume zoom working sessions to flesh this stuff out so it isn't a surprise like it was this time!

Reply:
No worries, appreciate the fixes here!